opm/simulators/wells/WellInterface.hpp

```diff
@@ -41,8 +41,11 @@
 namespace Opm {
     template<typename TypeTag> class GasLiftSingleWell;
     template<typename TypeTag> class BlackoilWellModel;
+    class GasLiftGroupInfo;
```

Comment:
why? you include it two line further down, nothing needs to know between those two lines.

Reply:
Thanks! I have removed the line.